README.rst

```diff
@@ -63,13 +65,27 @@ Settings list
 -------------
 All settings are contained in a ``PUSH_NOTIFICATIONS_SETTINGS`` dict.
 
-In order to use GCM, you are required to include ``GCM_API_KEY``.
-For APNS, you are required to include ``APNS_CERTIFICATE``.
+In order to use GCM, you are required to include one of ``GCM_API_KEY``, ``GCM_API_KEYS``, or ``GCM_API_KEYS_MODEL``.
+For APNS, you are required to include ``APNS_CERTIFICATE``, ``APNS_CERTIFICATES``, or ``APNS_CERTIFICATES_MODEL``.
 
 - ``APNS_CERTIFICATE``: Absolute path to your APNS certificate file. Certificates with passphrases are not supported.
+- ``APNS_CERTIFICATES``: A dictionary reflecting separate application IDs to separate APNS certificate files.
+- ``APNS_CERTIFICATES_MODEL``: A dictionary containing description of a database model to reflect
```

Comment:
Just to clarify, my use case is our business needs to add new `Application` instances as we develop partnerships with 3rd parties. We have a cookie cutter set of mobile apps (Android + iOS currently, maybe other platforms in future) that we whitelabel for them. 

Doing so means we need a different key/cert for each partnership - makes sense right?  The `Application` model is simple enough and doesn't need to be customized at all as long as it supports all the mobile platforms and can be extended in future via migrations.

I'm sure there are use cases where more complex modeling/logic is needed - it's just not my use case. :)